### PR TITLE
Implement new stairs fix stone stairs and slabs

### PR DESCRIPTION
**Pull request by IncredibleHolg**
Changes are rebased to recent version im master

### CODE REVIEW

**Comment by tmaas22** [comment]
Are you also going to add changes for the scaffolding and bamboo?

**Comment by IncredibleHolg** [comment]
@tmaas22 check out #1613.

**Comment by IncredibleHolg** [comment]
@CounterPillow did you get a notification about this pull request?

**Comment by CounterPillow** [comment]
I did, I'll get around to merging it when I have the time.

File: overviewer_core/textures.py [hunk 1]
```diff
@@ -1995,7 +2008,7 @@ def stairs(self, blockid, data):
     # sandstone, red sandstone, and quartz stairs have special top texture
     if blockid == 128:
         texture = self.load_image_texture("assets/minecraft/textures/block/sandstone_top.png").copy()
-    elif blockid == 156:
+    elif (blockid == 156 or blockid == 11375):
```
Comment (CounterPillow) [inline overviewer_core/textures.py]: Doesn't need parenthesis

File: overviewer_core/textures.py [hunk 2]
```diff
@@ -1995,7 +2008,7 @@ def stairs(self, blockid, data):
     # sandstone, red sandstone, and quartz stairs have special top texture
     if blockid == 128:
```
Comment (CounterPillow) [inline overviewer_core/textures.py]: Consider replacing this entire special case if block with a dictionary:

```python
special_tops = {
    128: "assets/minecraft/textures/block/sandstone_top.png",
    156: "assets/minecraft/textures/block/quartz_block_top.png",
    180: "assets/minecraft/textures/block/red_sandstone_top.png",
    11375: "assets/minecraft/textures/block/quartz_block_top.png",
}

if blockid in special_tops:
    texture = self.load_image_texture(special_tops[blockid]).copy()
```

This scales better if more blocks need to be added.

File: overviewer_core/textures.py [hunk 3]
```diff
@@ -1947,9 +1947,12 @@ def fire(self, blockid, data):
 # monster spawner
 block(blockid=52, top_image="assets/minecraft/textures/block/spawner.png", transparent=True)
 
-# wooden, cobblestone, red brick, stone brick, netherbrick, sandstone, spruce, birch, jungle, quartz, red sandstone, (dark) prismarine, mossy brick and mossy cobblestone stairs.
+# wooden, cobblestone, red brick, stone brick, netherbrick, sandstone, spruce, birch,
+# jungle, quartz, red sandstone, (dark) prismarine, mossy brick and mossy cobblestone, stone smooth_quartz
+# polished_granite polished_andesite polished_diorite granite diorite andesite end_stone_bricks red_nether_brick stairs
 @material(blockid=[53,67,108,109,114,128,134,135,136,156,163,164,180,203,11337,11338,11339,
-                   11370, 11371, 11374], data=list(range(128)), transparent=True, solid=True, nospawn=True)
+          11370, 11371,11374, 11375, 11376, 11377, 11378, 11379, 11380, 11381, 11382, 11383, 11384],
```
[the remark this answers is not in the document]
Comment (IncredibleHolg) [reply overviewer_core/textures.py]: @CounterPillow 
as soon as all changes are in, i will amend this to the pull request and notofy you

**Comment by IncredibleHolg** [comment]
@CounterPillow 
Your change request have been added, the pull request has been amend-updated.

**Pull request merged**